routemaster/subscription.rb

```diff
@@ -0,0 +1,22 @@
+module Routemaster
+  class Subscription
+
+    attr_reader :subscriber, :callback, :topics, :events
+
+    def initialize(options)
+      @subscriber = options.fetch('subscriber')
+      @callback   = options.fetch('callback')
+      @topics     = options.fetch('topics')
+      @events     = options.fetch('events').symbolize_keys
```

Comment:
Ah, got it. Thanks for adding the spec! 

It feels (very) ugly to me as well. I see two solutions then:
1) Implementing a naive version on the fly;
2) Extending the `Hash` class in here as well, just like we do in `routemaster`;
3) Deal with the `events` hash having string keys.

Thoughts?